facebook-for-woocommerce.php

```diff
@@ -71,7 +71,7 @@ public function wp_debug_display_error() {
       <p>
       <?php
         printf(__('To use Facebook for WooCommerce,
-          please disable WP_DEBUG_DISPLAY in your wp-config.php file.
+          please disable WP_DEBUG and WP_DEBUG_DISPLAY in your wp-config.php file.
```

Comment:
The notice, as you rightly identified in your comment, only occurs when both are on. 
So it's enough if one of them is disabled, so could you change this to `or` instead of `and` ?